includes/definitions/linux.yaml

```diff
@@ -34,3 +34,4 @@ register_mibs:
 discovery:
     - sysObjectID: .1.3.6.1.4.1.8072.3.2.10
     - sysDescr_regex: '/^Linux/'
+    - sysObjectID_except: .1.3.6.1.4.1.43296.
```

Comment:
You shouldn't need this here.

As your device is using `.1.3.6.1.4.1.43296.` for sysObjectID then it wouldn't match anyway.

Reply:
I was having issues here because Metamako devices have a sysDescr beginning with "Linux" though.